### DISCUSSION TITLE
在DX12游戏里用GDI捕捉会导致黑屏？

### DISCUSSION THREAD

**Discussion by 1732425146** [Blinue/Magpie#222] [Q&A]
DX11的游戏没有问题

**Comment by Blinue** [comment]
每种捕获方法都有优势和局限，GraphicsCapture 是兼容性最好的。DX12 因为实现问题无法通过常规方法捕获到，据我所知 GraphicsCapture 是唯一可用的方法（对性能不作要求的话 MagCallback 也可以）
